backend/.env.example

```
@@ -270,6 +270,14 @@ CONSENT_AUDIT_RETENTION_DAYS=365
 READINESS_CACHE_TTL_MS=2000
 
 # --- Observability ---
+# Enable OpenTelemetry tracing | boolean | # Optional (default: false)
+OTEL_ENABLED=false
+# OTLP collector endpoint for traces | URL | # Optional (default: http://localhost:4318/v1/traces)
+OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4318/v1/traces
+# OpenTelemetry service name | string | # Optional (default: stellar-portfolio-backend)
+OTEL_SERVICE_NAME=stellar-portfolio-backend
+# Enable Swagger API documentation UI | boolean | # Optional (default: true in dev, false in prod)
+ENABLE_API_DOCS=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ENABLE_API_DOCS key should go before the OTEL_ENABLED key

(UnorderedKey)